remote-backup/config.yaml

```diff
@@ -36,7 +36,7 @@ options:
   exclude_folders: ""
   exclude_addons: ""
   remote_directory: ""
-  zip_password: ""
+  password: ""

```

Comment:
I suggest calling it "backup_password" to clearly differentiate it from the user password needed for logging in.

Reply:
I agree, I have renamed it to `backup_password`